tests/test_physics.py

```diff
@@ -77,7 +80,9 @@ def test_kinematics(sim):
     old_object_id = sim.remove_object(object_id)
     assert len(sim.get_existing_object_ids()) == 0
 
-    object_id = sim.add_object(0)
+    obj_handle = sim.get_template_handle_by_ID(0)
```

Comment:
Let's unify all python tests for a known object using the "data/objects/cheezit" substring instead of ID->handle.

Reply:
are we sure it is always cheezit?  If so, we can just search on "chee" :).